sched/sched/sched_cpuload_period.c

```diff
@@ -183,6 +192,37 @@ static bool nxsched_period_callback(FAR uint32_t *next_interval_us,
   return true;
 }
 
+#ifdef CONFIG_PM
+static void nxsched_period_pmnotify(struct pm_callback_s *cb, int domain,
```

Comment:
```suggestion
static void nxsched_period_pmnotify(FAR struct pm_callback_s *cb, int domain,
```
`FAR` is required here, since this file is common to all targets.